Employee.java

```diff
@@ -0,0 +1,15 @@
+public class Employee {
+    int patientHealth;
+
+    public Employee() {
+        this.patientHealth = 10;
```

Comment:
should this be hardcoded?

Reply:
Yes, we need it set to 10 by default. We will make another constructor if we need it dynamic.

Reply:
Yes, blah blah blah